src/mod_commands.erl

```diff
@@ -407,12 +407,33 @@ delete_contact(Caller, Other) ->
     end.
 
 registered_commands() ->
+    Items = collect_commands(),
+    sort_commands(Items).
+
+sort_commands(Items) ->
+    WithKey = [{get_sorting_key(Item), Item} || Item <- Items],
+    Sorted = lists:keysort(1, WithKey),
+    [Item || {_Key, Item} <- Sorted].
+
+get_sorting_key(Item) ->
+    maps:get(path, Item).
+
+collect_commands() ->
     [#{name => mongoose_commands:name(C),
        category => mongoose_commands:category(C),
        action => mongoose_commands:action(C),
-       desc => mongoose_commands:desc(C)
+       method => mongoose_api_common:action_to_method(mongoose_commands:action(C)),
+       desc => mongoose_commands:desc(C),
+       args => format_args(mongoose_commands:args(C)),
+       path => mongoose_api_common:create_admin_url_path(C)
       } || C <- mongoose_commands:list(admin)].
 
+format_args(Args) ->
+    maps:from_list([{term_as_binary(Name), term_as_binary(Type)}
```

Comment:
The PR looks really nice. My only question is about `args`. When we have a string argument in a REST request, which is a binary internally, it will show up as a "binary" for the user, who lists the commands using this API. It could be confusing for them, as they are putting "string"s in their requests, right?

```
"args": {
      "jid": "binary",
      "caller": "binary"
    },
```
could rather be:
```
"args": {
      "jid": "string",
      "caller": "string"
    },
```